config/govuk_index/mapped_document_types.yaml

```diff
@@ -43,6 +43,7 @@ simple_smart_answer: edition
 statutory_instrument: statutory_instrument # Specialist Publisher
 special_route: edition
 step_by_step_nav: edition
+traffic_commissioner_regulatory_decision: traffic_commissioner_regulatory_decision # Specialist Publisher
```

Comment:
Can you also add this format to https://github.com/alphagov/search-api/blob/master/lib/learn_to_rank/format_enums.rb?

Reply:
Sure! I'll also do that for the new `marine_notice` format. 👍

Reply:
This has been done